scripts/new_version/new_version.sh

```diff
@@ -81,5 +79,7 @@ then
 
   $CURRENT_DIR/new_version_file.sh "wwi\/$old_version_without_revision\/" "wwi\/$new_version_without_revision\/" $WEBOTS_HOME/docs/dependencies.txt
 
+  $CURRENT_DIR/new_version_file.sh $old_version_without_revision $new_version_without_revision $WEBOTS_HOME/projects/humans/c3d/plugins/robot_windows/c3d_viewer_window/c3d_viewer_window.html
```

Comment:
This seems to be an unrelated change, why is it changed?

Reply:
It is better to change the version of files used in the c3d robot window only on major released.
So it is related because if run the script without fixing this the c3d robot window file will get updated.